website/docusaurus.config.js

```diff
@@ -104,22 +105,12 @@ module.exports = {
                     position: 'left',
                 },
                 {
-                    to: 'api',
-                    type: 'dropdown',
-                    label: versions[0],
-                    position: 'left',
-                    items: [
-                        { label: 'Next', to: 'api/next' },
-                        ...versions.map((version, i) => ({
-                            label: version,
-                            to: i === 0 ? 'api' : `api/${version}`,
-                            exact: true,
-                        })),
-                    ],
```

Comment:
this has to stay for now, we cant reuse the native version picker for API docs unfortunately, so we need both (and we will need to find a way to toggle between them dynamically so only one is visible most probably)

Reply:
> this has to stay for now, we cant reuse the native version picker for API docs unfortunately

Why?

Reply:
Because it does not respect the api docs at all